src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:19329/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:19329/project/issues?id=sample&branch=feature1&issues=AWFD5dhb5hqVUOc3Sc13&open=AWFD5dhb5hqVUOc3Sc13)

[ISSUE_KEY]: AWFD5dhb5hqVUOc3Sc13